kids-chat-app/.env.example

```
@@ -0,0 +1,24 @@
+# Database Configuration
+DATABASE_URL=postgresql://username:password@localhost:5432/kids_chat_db
+DATABASE_USER=your_db_username
+DATABASE_PASSWORD=your_db_password
+
+# NextAuth Configuration
+NEXTAUTH_SECRET=your_nextauth_secret_here
+NEXTAUTH_URL=http://localhost:3000
+
+# Admin Credentials
+ADMIN_USERNAME=admin
+ADMIN_PASSWORD=secure_admin_password_here
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The ADMIN_PASSWORD key should go before the ADMIN_USERNAME key

(UnorderedKey)